src/meta/src/rpc/metrics.rs

```diff
@@ -769,6 +779,16 @@ pub async fn start_fragment_info_monitor<S: MetaStore>(
                                     .mv_info
                                     .with_label_values(&[&mv_id_str, &table_id_str, &table.name])
                                     .set(1);
+                                if mv_id_str == table_id_str {
```

Comment:
Doing this judgement is because
- we can not directory use mv_info because it includes all tables(mv_table and it's internal_table), it's hard to `join`(aka `group left` in grafana) these two metrics.
- I do want to show materialized view name in this panel.

Reply:
Oh, I notice that  `const COLLECT_INTERVAL_SECONDS: u64 = 60;`, so I think the cost is tolerable.

Reply:
if we use mv_info: `many-to-many matching not allowed: matching labels must be unique on one side`

query: `sum(rate(stream_executor_row_count{executor_identity=~".*MaterializeExecutor.*", job=~"$job", instance=~"$node"}[$__rate_interval]) * on(actor_id) group_left(id) materialized_info) by (id)`

<img width="1563" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/58715567/338d9857-8fe7-451a-b926-bfd795c37467">

Reply:
I guess it's because one mv has many internal tables, thus cause many to many join.